### DISCUSSION TITLE
SMBX Wonder Edition?

### DISCUSSION THREAD

**Discussion by LoveBodhi** [Wohlstand/TheXTech#547] [Game Asset Pack]
It's just SMBX Asset, but with SMB Wonder styled gameplay system.

- Score and Timer (This doesn't exist in SMBX64) was disabled
- Combos are now rating (effect-79)
- Wonder FX which uses LUA script system

**Comment by Alucard648** [comment]
I don`t think, it`s any different than typical SMBX2 Asset pack. Wonder Flowers can be easily done with instant warp to subspace section, while all Wonder Effects are best made with LUA scripting.

**Reply by LoveBodhi** [reply]
So how does it work after wonder timer ends??

**Reply by LoveBodhi** [reply]
And fade in / out with customizable colours???

**Reply by LoveBodhi** [reply]
> Wonder Flowers can be easily done with instant warp to subspace section

Also can be done with Events...

**Comment by LoveBodhi** [comment]
Also, Score System is disabled in SMBW.

**Comment by LoveBodhi** [comment]
> I don`t think, it`s any different than typical SMBX2 Asset pack. Wonder Flowers can be easily done with instant warp to subspace section, while all Wonder Effects are best made with LUA scripting.

We're calling SMBX2 Asset Pack as `SMBX+`.

So we can consider to create a "Wonder Kit" library.

The function of WFX (Wonder Effects in short) can be like `onWonder()` or something.

**Comment by LoveBodhi** [comment]
We can select Wonder Mode to play with Riivolution-like options in the future, too.
当然了，我们也可以通过类似Riivolution的选项来选择惊奇模式。

**Comment by ds-sloth** [comment]
This would be possible to do using TheXTech engine after we add scripting support. What users make in asset packs is out of our control, but TheXTech is not intended to replicate any commercial IP. We take NO responsibility for the content made by users.
